engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1242,6 +1242,19 @@ public void testSkip() throws Exception {
 
   }
 
+  @Test
+  public void ise_184219679() throws IOException {
```

Comment:
ise?

Reply:
ise as a short for _"illegal state exception"_